webapp/src/components/DatasetWarnings/DatasetDistribution.tsx

```diff
@@ -9,13 +9,13 @@ import { DatasetWarningGroup } from "types/api";
 const DESCRIPTION = {
   "General Warnings": (
     <Description
-      text="See class distributions differences."
+      text="Investigate issues with class size, class imbalance and dataset shift."
       link="/dataset-warnings/#general-warnings"
     />
   ),
   "Syntactic Warnings": (
     <Description
-      text="Review the length of utterances between your training and test data."
+      text="Review the length of utterances between your training and evaluation data."
```

Comment:
Something feels awkward...I'm getting stuck here and am not sure which way to go, but here are some options:

```suggestion
      text="Review the length of utterances in your training and evaluation data."
```
or
```suggestion
      text="Review utterance lengths in your training and evaluation data."
```

or, if you want to highlight the difference between splits (and not just distribution within splits):

```suggestion
      text="Review any utterance length differences between your training and evaluation data."
```
or
```suggestion
      text="Compare utterance lengths for your training and evaluation data."
```